packages/block-editor/src/components/media-upload/index.native.js

```diff
@@ -147,6 +154,18 @@ export class MediaUpload extends Component {
 							source.types.includes( allowedType )
 					  );
 			} )
+			.filter( ( source ) => {
```

Comment:
Nitpick: What do you think about unifying both `filter` parts? Looks like we're filter based on the same value `source`.